app_dart/lib/src/service/scheduler.dart

```diff
@@ -322,17 +325,54 @@ class Scheduler {
       commitSha: commitSha,
       reason: reason,
     );
-    final Commit presubmitCommit = Commit(branch: branch, repository: slug.fullName, sha: commitSha);
-    final List<LuciBuilder> presubmitBuilders = await getPresubmitBuilders(
-      commit: presubmitCommit,
-      prNumber: prNumber,
-    );
-    await luciBuildService.scheduleTryBuilds(
-      builders: presubmitBuilders,
-      slug: slug,
-      prNumber: prNumber,
-      commitSha: commitSha,
+    final github.CheckRun ciValidationCheckRun = await githubChecksService.githubChecksUtil.createCheckRun(
+      config,
+      slug,
+      'ci.yaml validation',
+      commitSha,
     );
+    dynamic exception;
+    final Commit presubmitCommit = Commit(branch: branch, repository: slug.fullName, sha: commitSha);
+    try {
+      final List<LuciBuilder> presubmitBuilders = await getPresubmitBuilders(
+        commit: presubmitCommit,
+        prNumber: prNumber,
+      );
+      await luciBuildService.scheduleTryBuilds(
+        builders: presubmitBuilders,
+        slug: slug,
+        prNumber: prNumber,
+        commitSha: commitSha,
+      );
+    } catch (e) {
+      exception = e;
```

Comment:
Should we log this exception?

Reply:
we should also only catch a more specific type of Exception, rather than everything.

Reply:
https://dart.dev/guides/language/effective-dart/usage#avoid-catches-without-on-clauses

Reply:
I added a log.

We could catch `FormatException` and `HttpException` for ci.yaml related issues, but I figured if there's an issue in scheduling the LUCI builds (anywhere), we should mark it as failed. That way people will retry the presubmit instead of having a forever pending checkrun. WDYT?

Reply:
Moved to catch on FormatException. If another exception fails, this will leave the ci.yaml validation as pending.

Reply:
Will the exception bubbling up lead to a stronger signal that monitoring will detect? If not, then I suppose there's no advantage to disambiguating `FormatException`s.